tests/unit/metrics/test_transport.py

```diff
@@ -29,7 +29,7 @@
 # Some tests use real time! This is the time to wait between the exporter
 # thread handling tasks, and doesn't account for processing time. If these
 # tests become flaky, try increasing this.
-INTERVAL = .05
+INTERVAL = .1
```

Comment:
We may just want to scrub these tests or move them out of `unit/` if they're flaky for you.

Reply:
Changing to 0.1 seems to mitigate the problem. Not a big issue at this moment.